Sources/ServiceKit/Environment/Proccess.swift

```diff
@@ -0,0 +1,60 @@
+import Foundation
+
+extension Environment {
+    
+    /// The process information of an environment. Wraps `ProcessInto.processInfo`.
+    @dynamicMemberLookup public struct Process {
+        
+        /// The process information of the environment.
+        public let info: ProcessInfo
+        
+        /// Creates a new `Process` wrapper for process information.
+        ///
+        /// - parameter info: The process info that the wrapper accesses. Defaults to `ProcessInto.processInfo`.
+        public init(info: ProcessInfo = .processInfo) {
```

Comment:
This init could be kept private as well. The only way people should be creating one of these is via `Environment.process`.

Reply:
I had to make this internal so `Environment` could still initialize it.